xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,433 @@
+</screen>
+    <para>
+        This change tells the l3 agent that it must rely on the
+physnet-to-bridge mappings.
+    </para>
+</step>
+
+<step>
+    <para>
+    Restart <command>neutron-l3-agent</command>and 
+<command>neutron-openvswitch-agent</command>.
+</para>
+</step>
+
+<step xml:id="step.iface_maps">
+    <para>
+        Run either of the following two commands to set up interface mapping. 
+The first example is for Open vSwitch, and the second example is for 
+Linuxbridge users. These examples use another VLAN ID for the second 
+external network, so the interface mapping for Linuxbridge uses the same 
+network interface (eth0). It is enough to map the new network specifying the 
+correct segmentation ID.
+</para>
+    <screen>
+sudo neutron net-create public2 --provider:network_type flat \
+ --provider:physical_network public2 --router:external=True
+
+sudo neutron net-create public2 --router:external True \
+ --provider:physical_network physnet1 --provider:network_type vlan \
+ --provider:segmentation_id 600
+    </screen>
+</step>
+
+<step>
+    <para>
+        If a different network is used then &crow; will create a new interface 
+mapping. A flat network can be used then:
+</para>
+<screen>
+sudo neutron net-create public2 --provider:network_type flat \
+ --provider:physical_network public2 --router:external=True
+</screen>
+</step>
+
+<step>
+    <para>
+   Now create a subnet:     
+</para>
+<screen>
+sudo neutron subnet-create --name public2 --allocation-pool \
+ start=192.168.135.2,end=192.168.135.127 --gateway 192.168.135.1 public2 \
+ 192.168.135.0/24 --enable_dhcp False
+</screen>
+</step>
+
+<step>
+    <para>
+  The next step is to create a second router:  
+</para>
+<screen>
+sudo neutron router-create router2
+</screen>
+</step>
+
+<step>
+    <para>
+ Connect router2 to the second external network:       
+</para>
+<screen>
+sudo neutron router-gateway-set router2 public2
+</screen>
+</step>
+
+<step>
+    <para>
+        Create another private network and connect it to router2:
+</para>
+<screen>
+sudo neutron net-create priv-net
+sudo neutron subnet-create priv-net --gateway 10.10.10.1 10.10.10.0/24 \
+ --name priv-net-sub
+sudo neutron router-interface-add router2 priv-net-sub
+</screen>
+</step>
+
+<step>
+    <para>
+ Boot a VM setting the web security group and priv-net-sub network.       
+</para>
+</step>
+
+<step>
+    <para>
+ Assign a floating IP address to the VM, this time from network public2.
+</para>
+</step>
+
+<step>
+    <para>
+        From the node verify that SSH is working by opening an SSH session to 
+the VM.
+</para>
+</step>
+</procedure> 
+</sect1>
+
+<sect1 xml:id="sec.automatenet">
+    <title>Automating Network Setup</title>
+    <para>
+        Ideally, we need to be able to setup a second external network using 
+&crow;. This can be done as manual step, but this is error-prone.
+</para>
+
+<para>
+We should be able to handle the configuration file of the OVS agent that is on 
```

Comment:
"We should be able to..." displays a level of uncertainty that I don't think we want in out docs.